src/ray/core_worker/core_worker.cc

```diff
@@ -2711,45 +2711,54 @@ Status CoreWorker::CreateActor(const RayFunction &function,
       local_mode_named_actor_registry_.emplace(actor_name, actor_id);
     }
     ExecuteTaskLocalMode(task_spec);
+    return Status::OK();
+  }
+
+  task_manager_->AddPendingTask(
```

Comment:
The header comment for `AddPendingTask` needs to be updated: https://github.com/ray-project/ray/blob/c54437c42fa138580a0367f813b8c4bd9ca0b3e8/src/ray/core_worker/task_manager.h#L210